docs/notes.rst

```diff
@@ -1,8 +1,42 @@
-Version 1.8.1
+Version 1.8.2
 =============
 
 This release includes the following features:
 
+* Modified unknown writers default behavior.
+* Multiple initial PDP announcements.
+* Flag to avoid builtin multicast.
+* *STRICT_REALTIME* compilation flag.
+
+It also adds the following bug fixes and improvements:
+
+* Fix for setting `nullptr` in a fixed string.
+* Fix for not sending GAP in several cases.
+* Solve *Coverity* report issues.
+* Fix issue of *fastrtpsgen* failing to open *IDL.g4* file.
+* Fix unnamed lock in *AESGCMGMAC_KeyFactory.cpp*.
```

Comment:
*AESGCMGMAC_KeyFactory.cpp*... What?? Was this the name of the method with the unnamed lock?

Reply:
It is the name of the file.